pkg/vpc/vpc_test.go

```diff
@@ -25,21 +26,21 @@ type setSubnetsCase struct {
 }
 
 type importVPCCase struct {
-	Cfg               *api.ClusterConfig
+	cfg               *api.ClusterConfig
 	ID                string
```

Comment:
Oh, I meant all the fields in these test structs: ID, DescripveVPCOutput, etc. As far as I see these are used only in this file right?

Reply:
oh yeah, I was thinking about that after pushed changes, let me make another commit.

Reply:
another look if you don't mind, thanks :)